### DISCUSSION TITLE
[27 giugno 2023] Esercizio 3 (M-Z)

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/psd2223#254] [Esami]
Si realizzi l’interconnessione tra i registri sorgente $\Large R0,\ R1,\ R2$ e i registri destinazione $\Large D0,\ D1,\ D2$ e $\Large D3$ tale che:

- nel registro $\Large Di$, dove l’indice $\Large i$ è dato dai due bit meno significativi del registro $\Large R1$, viene trasferita la somma tra $\Large R0$ e $\Large R1$ se $\Large R0$ è pari, la differenza tra $\Large R1$ e $\Large R2$ altrimenti;

- il trasferimento è abilitato se $\Large R2 > R0$.

**Comment by Nw4pS** [comment]
@FeddyLix17 potresti postare la risoluzione dell'esercizio?

**Reply by FeddyLix17** [reply]
come per l'altra [richiesta](https://github.com/sapienzastudentsnetwork/progettazione-di-sistemi-digitali/discussions/265#discussioncomment-10606712), non essendo ufficiale ti consiglio di prendere la mia soluzione con le pinze

**Comment by FeddyLix17** [comment]
![photo_2024-09-11_23-40-32](https://github.com/user-attachments/assets/1f6b4cb6-a450-47f4-8373-030e230ea528)

**Reply by Nw4pS** [reply]
thx :3
